### DISCUSSION TITLE
제페토 월드 자체 제작한 아바타 애니메이션 재생

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#213] [5. Q&A]
안녕하세요.
유니티로 만든 제페토 월드에서 자체 제작한 아바타 애니메이션을 사용해도 되나요?
유니티 에셋의 아바타 애니메이션을 적용하려고 합니다.
가이드 부탁드립니다.

**Comment by x4cf** [comment]
혹시 도움이 될 수 있을까 싶어서 올려보아요 :)
https://github.com/naverz/zepeto-studio-kor/discussions/154

**Comment by moondory77-naverz** [comment]
안녕하세요.
해당 스레드 참고 부탁드리겠습니다.
[#65](https://github.com/naverz/zepeto-script-sample/issues/65#issuecomment-976678802)

감사합니다.

**Comment by strangestraw** [comment]
답변 감사드립니다!
